docs/clusters/README.md

```diff
@@ -0,0 +1,3 @@
+# Clusters
```

Comment:
Når dette har blitt merget inn så bør vi se på å lage bedre hovedsider for hver kategori.

Reply:
Ikke uenig, leker med å lage en egen mkdocs plugin som etteraper GitBook sin oppførsel (eller annen ønsket oppførsel) på fritiden.